Dockerfile

```diff
@@ -7,15 +7,74 @@ LABEL "com.github.actions.description"="Generates terraform code that calls terr
 LABEL "com.github.actions.icon"="activity"
 LABEL "com.github.actions.color"="blue"
 
+# Install the cloudposse alpine repository
 ADD https://apk.cloudposse.com/ops@cloudposse.com.rsa.pub /etc/apk/keys/
-RUN echo "@cloudposse https://apk.cloudposse.com/3.11/vendor" >> /etc/apk/repositories
+RUN echo "https://apk.cloudposse.com/3.12/vendor" >> /etc/apk/repositories
+RUN echo "https://alpine.global.ssl.fastly.net/alpine/edge/testing" >> /etc/apk/repositories
+RUN echo "https://alpine.global.ssl.fastly.net/alpine/edge/community" >> /etc/apk/repositories
 
-RUN apk add -u \
-				git \
-				variant2@cloudposse \
-				go-jsonnet@cloudposse==0.17.0-r0
+RUN apk add -u git
+RUN wget https://github.com/mumoshu/variant2/releases/download/v0.37.1/variant_0.37.1_linux_amd64.tar.gz -O /tmp/var.tgz && \
+    tar xzf /tmp/var.tgz && \
+	rm /tmp/var.tgz && \
+	mv variant /usr/local/bin
+RUN wget https://github.com/google/go-jsonnet/releases/download/v0.17.0/go-jsonnet_0.17.0_Linux_x86_64.tar.gz -O /tmp/jsonnet.tgz && \
+    tar xzf /tmp/jsonnet.tgz && \
+	rm /tmp/jsonnet.tgz && \
+	mv jsonnet /usr/local/bin
 
-# Build the CLI
 WORKDIR /usr/cli
+
+FROM base as run
+# Build the CLI
 COPY ./ ./
 ENTRYPOINT [ "/usr/cli/main.variant", "--install_dir", "/usr/cli" ]
+
+FROM base as test
```

Comment:
Note, I moved the Dockerfile.test to here because `docker build` wasn't caching results. Feel free to find other ways to solve that problem. I just did something so I wasn't waiting on a buncha curls

Reply:
(in the future suggest `git mv` to preserve the history of the file and reduce diff)